devices/philips.js

```diff
@@ -1525,6 +1525,15 @@ module.exports = [
         extend: hueExtend.light_onoff_brightness_colortemp_color(),
         ota: ota.zigbeeOTA,
     },
+    {
+        zigbeeModel: [ 'LCF003', '4080248P7' ],
+        model: '4080248P7',
+        vendor: 'Philips',
+        description: 'Hue Signe floor light',
+        meta: { turnsOffAtBrightness1: true },
+        extend: hueExtend.light_onoff_brightness_colortemp_color(),
```

Comment:
Please add the color temperature range as described on https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html#_3-adding-converter-s-for-your-device

Reply:
added